Wrappers/Python/cil/framework/BlockGeometry.py

```diff
@@ -49,9 +49,8 @@ def get_item(self, index):
 
     def allocate(self, value=0, dimension_labels=None, **kwargs):
         
-        max_value = kwargs.get('max_value', 100)
         symmetry = kwargs.get('symmetry',False)        
-        containers = [geom.allocate(value, max_value = max_value) for geom in self.geometries]
+        containers = [geom.allocate(value, **kwargs) for geom in self.geometries]

```

Comment:
max_value is a `kwarg` used in the `allocate` method of the `ImageGeometry`, or `VectorGeometry` or `AcquisitionGeometry`

Reply:
yes, but before it was (undocumented) with a default value of 100. Now what will it be? 1?

Reply:
it will be the default kwarg argument pass from the allocate method of the ImageGeometry, which by default is [100](https://github.com/TomographicImaging/CIL/blob/master/Wrappers/Python/cil/framework/framework.py#L315)

Reply:
it is in master, not in this branch where line 52 is removed